src/controllers/orderController.js

```diff
@@ -0,0 +1,25 @@
+const { isEmpty } = require('lodash');
+const { throwError } = require('../utils');
+const { getSeatsDataService, isEventExistService } =
+  require('../services').orderService;
+
+const getSeatsController = async (req, res, next) => {
+  try {
+    const { eventId } = req.query;
+    if (!eventId)
+      throwError(400, 'required query parameter eventId is missing');
+    const eventExist = await isEventExistService(eventId);
+    if (isEmpty(eventExist)) throwError(400, 'no event data');
+    const seatsData = await getSeatsDataService(eventId);
```

Comment:
```suggestion
    const seatsData = await getSeatsDataByEvent(eventId);
```
1. 레이어의 모든 함수이름 뒤에 `service`, `controller`, `dao` 단어가 붙어있는데, 팀내 다른 분들 함수에는 안붙어 있어서, 정해진 컨벤션이 없다면 맞추시는게 좋을 것 같습니다-!
2. 이벤트로 걸러서 데이터를 추출하는 과정이므로, 어떤 entity의 id를 넘기는 지 함수명에서 명시하시는 것이 좋습니다.

Reply:
원래는 컨벤션이 맞으나 잘 지켜지지 않은 것 같습니다 🥲 이름을 바꿔야 한다면 단순히 dao를 리턴하는 service의 경우 이름 짓기가 모호한 감이 있어서 접미사로 구분해야 할 것 같습니다